python/lsst/ts/xml/data/sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -494,4 +494,82 @@
       <Count>1</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <EFDB_Topic>ESS_electricFieldStrength</EFDB_Topic>
+    <Description>Electric field strength for a period of 10 seconds (20 measurements per second).</Description>
+    <item>
+      <EFDB_Name>sensorName</EFDB_Name>
+      <Description>The user-defined name of the sensor (e.g. AuxTel EFM-100C).</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>Starting time of data acqusition (TAI Unix seconds) used to measure EFSs.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>strength</EFDB_Name>
+      <Description>Strength of the electric field (range -20.00 to 20.00).</Description>
```

Comment:
Please consider publishing max, mean or median, and std dev or quartiles.

If you want to just publish the one value please explain what it is -- max? mean?

I suggest having the measure in the name, e.g. `strengthMax` (and `strengthStdDev` if you add them)